containers/datalab/content/setup-env.sh

```diff
@@ -25,3 +25,7 @@ then
   export CLOUDSDK_CONFIG=/content/datalab/.config
 fi
 
+if [ -n "${EXPERIMENTAL_KERNEL_GATEWAY_URL}" ]
+then
+  export KG_URL="${EXPERIMENTAL_KERNEL_GATEWAY_URL}"
```

Comment:
Do we have to use two env var (EXPERIMENTAL_KERNEL_GATEWAY_URL and KG_URL) here? Can we just use EXPERIMENTAL_KERNEL_GATEWAY_URL?